library/tasks/ScatterBam.wdl

```diff
@@ -0,0 +1,27 @@
+task ScatterBam {
+
+  File bam_to_scatter
+  Int scatter_width
+  Int disk_size = ceil(size(bam_to_scatter, "GB") * 3)
+
+  command <<<
+    mkdir scattered_bams
+    java -Xms7g -jar /usr/gitc/picard.jar \
+      SplitSamByNumberOfReads \
+      INPUT=${bam_to_scatter} \
+      SPLIT_TO_N_FILES=${scatter_width} \
+      OUT_PREFIX=$(basename ${bam_to_scatter} .bam)_split \
+      OUTPUT=scattered_bams
+  >>>
+
+  output {
+    Array[File] scattered_bams = glob("scattered_bams/*")
+  }
+
+  runtime {
+    disk: "local-disk ${disk_size} HDD"
+    cpu: 2
+    memory: "7.5 GB"
```

Comment:
Should this be GiB or not yet?

Reply:
Personal preference: style should always be consistent across the codebase. Right now we use `GB`. If we want or need to change to `GiB`, that should be done as a single refactor that updates the entire codebase of skylab, not in individual PRs (which would cause us to have both `GB` and `GiB` simultaneously).

Reply:
Is Optimus running on PAPIv2?

Reply:
Yes, I think all HCA pipelines are running on PAPIv2: https://github.com/HumanCellAtlas/pipeline-tools/blob/master/adapter_pipelines/Optimus/options.json

Reply:
Then it should be GiB. I'll change it.

Reply:
Do we have any pipelines that are run on PAPIv1?

If not, I would prefer someone to go through our codebase and make the relevant `s/GB/GiB` refactor. It is confusing to have two styles present simultaneously.

Reply:
We changed all the tasks used by SS2 and Optimus to use GiB in [this PR](https://github.com/HumanCellAtlas/skylab/pull/189) to fix out of memory issues. We can put in a ticket to change the rest of the tasks in Skylab